Cargo.toml

```diff
@@ -24,8 +25,11 @@ doctest = false
 crate-git-revision = "0.0.6"
 
 [dev-dependencies]
-proptest = "1.0.0"
+proptest ="1.0.0"
+serde = { version = "1.0.214", features = ["derive"] }
+toml = "0.8.19"
```

Comment:
We can use `serde_test` instead of toml, to test the serde impl specifically rather than test it via toml, json, etc.